components/About/MeetOurTeam.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Container } from "react-bootstrap";
+import { useTranslation } from "next-i18next";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+
+interface Box {
+    title: string
+    number: string
+    subtext: string
+}
+
+const MeetOurTeam:React.FC = () => {
+
+    const { t } = useTranslation("about");
+    const contentArray = [
+        t("content.meet.content1", {returnObjects: true}),
+        t("content.meet.content2", {returnObjects: true}),
```

Comment:
since you loop through them in `MeetOurTeam.tsx`, perhaps in `about.json` it should match?
```
"team": {
        "title": "Meet our team",
        "cardDetails": [
            {
                "pre": "Written by",
                "statistic": "35+",
                "post": "Missionaries & Japanese Christian leaders"
            }, {
                "pre": "A collaboration from",
                "statistic": "17",
                "post": "Organizations"
            }, {
                "pre": "This website is created and built by",
                "statistic": "9+",
                "post": "Servants of the Church"
            }
        ]
    }
```